packages/atomic/src/components/facets-v1/atomic-rating-facet/atomic-rating-range-facet.tsx

```diff
@@ -0,0 +1,244 @@
+
+  /**
+   * Specifies a unique identifier for the facet.
+   */
+  @Prop({mutable: true, reflect: true}) public facetId?: string;
+  /**
+   * The non-localized label for the facet.
+   */
+  @Prop() public label = 'no-label';
+  /**
+   * The field whose values you want to display in the facet.
+   */
+  @Prop() public field!: string;
+  /**
+   * The number of intervals to split the index for this facet.
+   */
+  @Prop() public numberOfIntervals = 5;
+  /**
+   * The maximum value of the field. This value is also used as the number of icons to be displayed.
+   */
+  @Prop() public maxValueInIndex = this.numberOfIntervals;
+  /**
+   * The icon used to display the rating.
+   */
+  @Prop() public icon = Star;
+
+  public initialize() {
+    this.searchStatus = buildSearchStatus(this.bindings.engine);
+    this.initializeFacet();
+  }
+
+  private initializeFacet() {
+    const options: NumericFacetOptions = {
+      facetId: this.facetId,
+      field: this.field,
+      numberOfValues: this.numberOfIntervals,
+      currentValues: this.generateCurrentValues(),
+      sortCriteria: 'descending',
+      generateAutomaticRanges: false,
+    };
+    this.facet = buildNumericFacet(this.bindings.engine, {options});
+    this.facetId = this.facet.state.facetId;
+    this.bindings.store.state.numericFacets[this.facetId] = {
+      label: this.label,
+      format: (facetValue) => this.formatFacetValue(facetValue),
+    };
+  }
+
+  private get scaleFactor() {
+    return this.maxValueInIndex / this.numberOfIntervals;
+  }
+
+  private get numberOfSelectedValues() {
+    return this.facetState.values.filter(({state}) => state === 'selected')
+      .length;
+  }
+
+  private generateCurrentValues() {
+    const currentValues: NumericRangeRequest[] = [];
+    for (let i = 0; i < this.numberOfIntervals; i++) {
+      currentValues.push(
+        buildNumericRange({
+          start: Math.round((i + 1) * this.scaleFactor * 100) / 100,
+          end: Math.round(this.maxValueInIndex * 100) / 100,
+          endInclusive: true,
+        })
+      );
+    }
+    return currentValues;
+  }
+
+  private formatFacetValue(facetValue: NumericFacetValue) {
+    return this.bindings.i18n.t('to', {
+      start: facetValue.start,
+      end: facetValue.end,
+    });
+  }
+
+  private renderHeader() {
+    return (
+      <FacetHeader
+        i18n={this.bindings.i18n}
+        label={this.label}
+        onClearFilters={() => this.facet.deselectAll()}
+        numberOfSelectedValues={this.numberOfSelectedValues}
+        isCollapsed={this.isCollapsed}
+        onToggleCollapse={() => (this.isCollapsed = !this.isCollapsed)}
+      ></FacetHeader>
+    );
+  }
+
+  private renderLabelText(facetValue: NumericFacetValue) {
+    return (
+      <span class="ml-1 flex items-center group-hover:underline group-hover:text-primary">
+        {facetValue.start === this.maxValueInIndex ? (
+          <span class="font-bold">only</span>
```

Comment:
`only` & `& up` have to be internationalized using `this.bindings.i18n.t(...)`

Reply:
It is bold then selected (any value) & not bold when idle

Reply:
The labels should have the part `value-label`

Reply:
What does the value looks like when the labels are very long, does it overflow?

Reply:
after adding the `truncate` class to the parent span:
![Screen Shot 2021-07-23 at 2 00 41 PM](https://user-images.githubusercontent.com/83350303/126822804-7cc8a9ac-9075-4645-adac-1387bc35b692.png)

Reply:
bold when selected:
![Screen Shot 2021-07-23 at 2 15 01 PM](https://user-images.githubusercontent.com/83350303/126824255-124fa690-6a4b-4a64-9af2-393344a2a8d8.png)